src/content/docs/dev/general-message-passing/xrpl/xrpl-gmp.mdx

```diff
@@ -0,0 +1,196 @@
+# XRPL General Message Passing (GMP)
+
+The XRPL Blockchain is unique compared to other blockchains integrated with Axelar. Unlike other blockchains, XRPL does not support smart contracts. As such there is no Axelar Gateway and Axelar Gas Service deployed on the chain the way there is with other EVM and non-EVM blockchain integrations. Since XRPL has no contracts there is also no ability to receive GMP messages from other blockchains, you can only send messages to other blockchains.
+
+## XRPL Multisig
+
+The integration leverages [XRPL Multisig Signing](https://xrpl.org/docs/concepts/accounts/multi-signing) to facilitate message passing. The [multisig](https://github.com/commonprefix/axelar-amplifier/blob/xrpl/contracts/xrpl-multisig-prover/src/xrpl_multisig.rs) is controlled by the [Axelar Verifier set](https://axelarscan.io/verifiers).  
```

Comment:
```suggestion
The integration leverages [XRPL Multisig Signing](https://xrpl.org/docs/concepts/accounts/multi-signing) to transfers to XRPL. The [multisig](https://github.com/commonprefix/axelar-amplifier/blob/xrpl/contracts/xrpl-multisig-prover/src/xrpl_multisig.rs) is controlled by the [Axelar Verifier set](https://axelarscan.io/verifiers).  
```

We don't really use multisigning for message passing from XRPL, it's just RPC voting like other connections.

Reply:
this is for messages coming in/out of xrpl no? not just one way.